src/sdk/pynni/nni/__main__.py

```diff
@@ -139,17 +105,14 @@ def main():
             raise
 
 
-def _run_advisor(args):
-    if args.advisor_class_name in AdvisorModuleName:
+def _run_advisor(exp_params):
+    if exp_params.get('advisor').get('builtinAdvisorName') in AdvisorModuleName:
```

Comment:
If `exp_params.get('advisor')` can be None, I think `get` won't help. Should write `exp_params['advisor']` anyway.

Reply:
Here the key 'advisor' must exist, it is OK to use either `exp_params['advisor']` or `exp_params.get('advisor')`, just want to be consistent with other places where key may not exists.